bin/anthology/volumes.py

```diff
@@ -69,8 +69,6 @@ def from_xml(volume_xml,
         front_matter_xml = volume_xml.find('frontmatter')
         if front_matter_xml is not None:
             front_matter = Paper.from_xml(front_matter_xml, volume, formatter)
-            front_matter._id = '0'
-            front_matter.is_volume = True
```

Comment:
Note: this isn't directly related to the bug, but it's a potential source of future bugs since it doesn't do what it seems to do. (The values of `_id` and `is_volume` are already important during the `Paper.from_xml` call.)